app/controllers/fhir/r4/api_controller.rb

```diff
@@ -1130,6 +1017,38 @@ def format_fhir_validation_errors(errors)
     end
   end
 
+  def patient_read_scopes
+    %i[user/Patient.read user/Patient.* system/Patient.read system/Patient.*]
+  end
```

Comment:
Can these arrays be constants rather than methods?

Reply:
To constants in https://github.com/SaraAlert/SaraAlert/pull/843/commits/a320e4de33bb0b35fd5555de09f6e987e417ad1d.

Reply:
Would you mind just moving them to the top of the class? I'd probably put them right after the includes.

Reply:
To the top in https://github.com/SaraAlert/SaraAlert/pull/843/commits/311e4005ae1ca979bf308abcbad38d7bf99859ef.